keyboards/anavi/macropad12/config.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 Leon Anavi <leon@anavi.org>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define RGBLIGHT_DEFAULT_MODE RGBLIGHT_MODE_RAINBOW_MOOD
+
+/* Double tap reset button to enter bootloader */
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP17
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 500U
+
+#define BACKLIGHT_PWM_DRIVER PWMD5
+#define BACKLIGHT_PWM_CHANNEL RP2040_PWM_CHANNEL_A
```

Comment:
Entirely optional, matches logic of lines 19-23
```suggestion
#ifdef BACKLIGHT_ENABLE
#    define BACKLIGHT_PWM_DRIVER PWMD5
#    define BACKLIGHT_PWM_CHANNEL RP2040_PWM_CHANNEL_A
#endif
```

Reply:
Yes, this will be better for the consistency of the file. Thanks!